vuepress/docs/docs/federation.md

```diff
@@ -25,7 +25,7 @@ interaction with the gateway.
 If you already have a graph you can add federation simply by adding the `federated` annotation:
 
 ```scala
-import caliban.federation._
+import caliban.federation.v1._
```

Comment:
So we need to mention in the release notes that this is necessary, right? There is no more "default" version.

Reply:
Yep, I suppose I could add another constant `v2` pointed at `v2_0` and then deprecate it as well? Though I'd need to move the structure around a bit since it would conflict with the current `v2` package

Reply:
Oh actually misread what you were saying, correct there is no more default version since there is so much difference between the latest versions of federation I think it makes sense to make that explicit

Reply:
> Yep, I suppose I could add another constant v2 pointed at v2_0 and then deprecate it as well

I was going to recommend that at first, but then I realised there's a `v2` package already, so I thought it wouldn't be possible to define a `v2` constant. But if it is possible, I think it's a good idea

Reply:
I moved the `v2` package to `v2x` instead and then made a new deprecated `v2` constant that points at `v2_0`. It definitely isn't bincompat but it should at least still be source compatible for the most part.